lib/mi_bridges/driver.rb

```diff
@@ -86,15 +87,27 @@ def run
 
     private
 
-    attr_reader :snap_application
+    def logger=(logger)
```

Comment:
seems like we defined almost the same method in `Export.rb` -- should it be extracted?

Reply:
@jessieay - Yea, the seams aren't quite there, and since there are only two and not three I kicked it back.

Reply:
Wound up extracting a LoggerFactory, but I am somewhat filled with shame for it.